server/datastore/mysql/migrations/tables/20240909145426_AddFleetLibraryAppsTable.go

```diff
@@ -0,0 +1,69 @@
+package tables
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20240909145426, Down_20240909145426)
+}
+
+func Up_20240909145426(tx *sql.Tx) error {
+	_, err := tx.Exec(`
+CREATE TABLE fleet_library_apps (
+	id               int unsigned NOT NULL PRIMARY KEY AUTO_INCREMENT,
+	name             varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
+	-- the "full_token" field from homebrew's JSON API response
+	-- see e.g. https://formulae.brew.sh/api/cask/1password.json
+	token            varchar(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci NOT NULL,
```

Comment:
Interestingly, "formulas" don't seem to have that token/full_token field (e.g. : https://formulae.brew.sh/api/formula/redis.json), but my guess is that we'll package "casks" like the 1password example.